app/models/hiring.rb

```diff
@@ -3,6 +3,15 @@ class Hiring < ApplicationRecord
 
   belongs_to :user
   belongs_to :vehicle
+  belongs_to :status
 
   validates :time, presence: true
+
+  def self.search search
+    if search
+      Hiring.where id: search
```

Comment:
params search là list các id ah? search theo mỗi id thôi ah e?

Reply:
Theo kịch bản thì người dùng mang mail đến cửa hàng để nhận xe, nhân viên sẽ search id theo id được in trên email ạ, nên chỗ này chỉ cần search theo email thôi ạ

Reply:
viết scope đi e :)